runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -827,8 +827,9 @@ class CompilationInfo
    void     setAotRelocationTime(uint32_t reloTime) { _statTotalAotRelocationTime = reloTime; }
    void    incrementNumMethodsFoundInSharedCache() { _numMethodsFoundInSharedCache++; }
    int32_t numMethodsFoundInSharedCache() { return _numMethodsFoundInSharedCache; }
+#if defined(JITSERVER_SUPPORT)
```

Comment:
Is this still needed? Maybe until Harry ports the changes from Younes

Reply:
If this PR is merged before #7344, `isRomClassForMethodInSharedCache` should be kept. #7344 should rebase first. If this PR is merged after, I'll rebase this PR to the latest `jitaas` and remove `isRomClassForMethodInSharedCache`.